images/scripts/arch.setup

```diff
@@ -78,8 +78,12 @@ systemctl enable --now firewalld
 # Open firewall for cockpit
 firewall-cmd --permanent --zone=public --add-port 9090/tcp
 
-# Enable networkmanager, as no services are started by default
+# Enable networkmanager, disable systemd-networkd as no services are started by default
 systemctl enable NetworkManager
+systemctl disable systemd-networkd
```

Comment:
Is that really required? Ubuntu uses systemd-networkd as well to manage its primary eth0 interface, and that should not get in the way at all.

Reply:
hmm now you make me wonder if I should not disable it, and let networkmanager ignore eth0 as well.... Would that maybe fix some of the test issues we have with networkmanager at the moment? Such as

https://logs.cockpit-project.org/logs/pull-16547-20211116-111534-39d11fea-arch-bots-2619/log.html#34-2

Reply:
It doesn't, seems to be some issue with Arch's networkmanager.